detector/rules.go

```diff
@@ -68,12 +68,12 @@ func LoadRules(path string) (*Rules, error) {
 	}
 
 	rules := &Rules{
-		WhiteList:  make(map[string]struct{}, len(rf.Whitelist)),
+		AllowList:  make(map[string]struct{}, len(rf.Allowlist)),
 		YellowList: make(map[string]struct{}, len(rf.Yellowlist)),
```

Comment:
Might as well change this too to something like `maybelist` for consistency.

Reply:
The intention of the yellowlist here is a bit confusing for me, since we do not treat it any differently so to me it seems like it should all just be an allowlist, and users can decide if their usage of different licences meets the requirements of the allowlist. Am I missing something?

For reference, internally they use green/yellow/red lists.

Reply:
This came from a PR from the Beat team. I think the idea is to highlight the fact that there are some contentious licences in the repo. If they all went into the allow list, they could go unnoticed.

I am fine with whatever terminology that is consistent. Allow list and yellow list, as it is now,  doesn't make sense to me.